plutus-core/plutus-core/src/PlutusCore/Subst.hs

```diff
@@ -161,6 +164,56 @@ termSubstClosedTerm varFor new = go where
          t                    -> t & over termSubterms go
     goUnder var term = if var == varFor then term else go term
 
+-- Mapping names
```

Comment:
This is so that the `plc` and `uplc` commands can perform the same conversions. It wasn't too hard to implement so I thought it wouldn't do any harm to have it since we already `flat` instances for the different types of names.  It's a different story for PIR though.